commands/entity-store.ts

```diff
@@ -74,30 +78,32 @@ const ingestEvents = async (events) => {
   for (let chunk of chunks) {
     try {
       // Make bulk request
-      let ingestRequest = chunk.reduce((acc, event) => {
+      let ingestRequest = chunk.reduce((acc: (BulkOperationContainer | BulkUpdateAction<TDocument, TPartialDocument> | TDocument)[], event) => {
```

Comment:
typing this array to be the type that the bulk client takes. This isn't pretty, but it passes :P